docs/managing-dragonfly/acl.md

```diff
@@ -0,0 +1,127 @@
+active and authenticated connections.
+
+Also note, that the flag `--requirepass` now also changes the `default` user password. So, if during Dragonfly startup the flag `requirepass` is set,
+then the `default` user's password will be the one specified in that flag.
+
+## ACL Groups
+Each `command` belongs to a set of ACL groups. The syntax for specifying a group is:
+
+```
++@GROUP_NAME
+-@GROUP_NAME
+```
+
+The sign at the front dictates the operation (`+` for granting and `-` for revoking). The `@` denotes category (ACL group) and
+the `GROUP_NAME` is the name of the group. For example:
+
+```
+ACL SETUSER John ON +@FAST
+```
+
+Updates the permissions of user `John` and grants him the ability to run any command in the group `FAST`. Revoking this,
+is straighforward:
+
+```
+ACL SETUSER John -@FAST
+```
+
+There are also the special keywords `+@ALL, -@ALL` that allow the user to run commands found in `ALL GROUPS`.
+
+With `@ALL` it's possible to do:
+
+```
+ACL SETUSER John +@ALL -@ADMIN -@FAST
+```
+
+Which basically grants all but the `@ADMIN` and `@FAST` groups to the user. That way, it's easy to express which groups of permissions
+the user should not be a part of.
+
+## Commands
+
+Dividing the commands into groups offers a great flexibility of quickly granting/revoking permissions but it's somehow limited because
+these groups are not user defined. Therefore, for finer control, the user can specify a list of commands that is explicitly allowed to execute.
+For example:
+
+```
+ACL SETUSER John +GET +SET +@FAST
+```
+
+This allows the user `John` to execute only the `SET` && `GET` commands and all of the commands associated with the group `FAST`.
+Any attempt of user `John` to issue a command other than the above, will be rejected by the system.
+
+Note that the syntax is similar to the ACL groups, but without the prefix `@`.
+
+The special `+ALL` (note without the `@`) is used to denote all of the currently implemented commands.
+
+## Persistence
+
+The state of all of the users and their permissions can be captured and placed in a file. As with redis,
+the `--aclfile` option is used to specify the file from which Dragonfly will load the ACL state from.
+
+Afterwards, any change done at runtime, can be persisted at anytime via the command `ACL SAVE` which
+evicts the currently stored ACL state to the file specified in the `--aclfile` option.
+
+Note, that the `aclfile` file is compatible with Redis (however it must not contain any keys or
+pub/sub DSL's because these yet are not supported so if you plan to migrate, just open the file and strip them away).
+
+If you want the `aclfile` to be writable, that is, if you want `ACL SAVE` to work, we would advice against placing the `aclfile`
+under `/etc` directory because the folder is only accesible by Dragonfly as `readonly`. You change this behaviour, by editing 
```

Comment:
```suggestion
under `/etc` directory because usually that directory is only accessible by Dragonfly as `readonly`. You change this behavior, by editing 
```